packages/browser/src/posthog-surveys.ts

```diff
@@ -327,6 +328,32 @@ export class PostHogSurveys {
             logger.warn('Survey not found')
             return
         }
+        if (!IN_APP_SURVEY_TYPES.includes(survey.type)) {
+            logger.warn(`Surveys of type ${survey.type} are cannot be rendered in the app`)
+            return
+        }
+        if (!elem) {
+            logger.warn('Survey element not found')
+            return
+        }
+        this._surveyManager.renderSurvey(survey, elem)
+    }
+
+    private _renderExternalSurvey(surveyId: string, selector: string) {
```

Comment:
you can extract this method and renderSurvey to another method that takes a `type` list as a param, this method passes `[SurveyType.ExternalSurvey]` and the other one `IN_APP_SURVEY_TYPES`, so you dont have to duplicate the whole method body twice next to each other because of one single condition

Reply:
great idea, just did it